packages/squiggle-lang/src/fr/plot.ts

```diff
@@ -46,11 +48,33 @@ export const library = [
           });
           return Result.Ok(
             vPlot({
+              type: "distributions",
               distributions,
             })
           );
         }
       ),
     ],
   }),
+  maker.make({
+    name: "fn",
+    output: "Plot",
+    examples: [`Plot.fn({fn: {|x|x*x}, min: 3, max: 5})`],
+    definitions: [
+      makeDefinition(
+        "fn",
+        [frRecord(["fn", frLambda], ["min", frNumber], ["max", frNumber])],
```

Comment:
you might want to add to Documentation, I assume that's not automatically done right now.